backend/module/dynamodb/dynamodb.go

```diff
@@ -30,7 +30,12 @@ func (a *dynamodbAPI) DescribeTable(ctx context.Context, req *dynamodbv1.Describ
 }
 
 func (a *dynamodbAPI) UpdateTableCapacity(ctx context.Context, req *dynamodbv1.UpdateTableCapacityRequest) (*dynamodbv1.UpdateTableCapacityResponse, error) {
-	return nil, status.Error(codes.Unimplemented, "not implemented")
+	err := a.client.UpdateTableCapacity(ctx, req.Region, req.TableName, req.TargetTableRcu, req.TargetTableWcu)
+	if err != nil {
+		return nil, err
+	}
+
+	return &dynamodbv1.UpdateTableCapacityResponse{}, nil
```

Comment:
Is the response going to echo back the table changes?

Reply:
Yup it'll echo back the table name and the target capacities

Reply:
We still going to do the echo back here? Or will that come in a later PR?

Reply:
I'll add in a separate PR since I'll also have to add the echo fields to the [dynamodb api proto](https://github.com/lyft/clutch/blob/0a6e4a00e72b8bb3a976f801d02f9550adf447d7/api/aws/dynamodb/v1/dynamodb.proto#L78) too right?

Reply:
Ah yeah that sounds good, I assumed they were already on the proto.